src/utils/api.ts

```diff
@@ -1,4 +1,10 @@
-export const api = async ({ url, method, body }) => {
+interface apiGet {
```

Comment:
this isn't only for get, get doesnt allow for a body. this is just the general request interface

Reply:
body is optional

Reply:
<img width="429" alt="image" src="https://github.com/user-attachments/assets/2e7c84d4-f750-48ad-a3d9-59e9adfdd832" />
use this type